src/components/NavBar.vue

```diff
@@ -29,6 +29,16 @@
       />
       <v-spacer />
 
+      <amb-button
+        with-badge
+        data-cy="timeline-icon"
+        is-icon
+        icon="history"
+        btn-color="secondary"
+        v-if="!isEdgeConnected"
```

Comment:
@vickywane the flag `v-if="!isEdgeConnected"` was copied verbatim from the `download-off` icon, but that's not the correct behavior. This makes the timeline icon disappear when there *is* connection and appear only when there is *no* connection. Should be the opposite show/hide behavior of the `download-off` icon.

Please fix and add tests to cover these scenarios. Thank you!